controller/backup_target_controller.go

```diff
@@ -414,6 +430,67 @@ func (btc *BackupTargetController) reconcile(name string) (err error) {
 	return nil
 }
 
+func (btc *BackupTargetController) syncSystemBackup(backupTargetClient *engineapi.BackupTargetClient, log logrus.FieldLogger) error {
+	backupstoreSystemBackups, err := backupTargetClient.SystemBackupList(backupTargetClient.URL, backupTargetClient.Credential)
+	if err != nil {
+		return errors.Wrapf(err, "failed to list system backups in %v", backupTargetClient.URL)
+	}
+
+	clusterSystemBackups, err := btc.ds.ListSystemBackups()
+	if err != nil {
+		return errors.Wrap(err, "failed to list SystemBackups")
+	}
+
+	clusterReadySystemBackupNames := sets.NewString()
+	for _, systemBackup := range clusterSystemBackups {
+		if systemBackup.Status.State != longhorn.SystemBackupStateReady {
+			continue
+		}
```

Comment:
Is this to avoid syncing the in-progress SystemBackups handled by other clusters? If YES, you can add a comment

Reply:
This is to avoid deleting the in-progress SystemBackup.